libopenage/coord/phys.h

```diff
@@ -1,13 +1,16 @@
-// Copyright 2016-2018 the openage authors. See copying.md for legal info.
+// Copyright 2016-2019 the openage authors. See copying.md for legal info.
 
 #pragma once
 
 #include "../util/misc.h"
+#include "../util/hash.h"
 
 #include "declarations.h"
 #include "coord_nese.gen.h"
 #include "coord_neseup.gen.h"
 
+#include <typeindex>
+
```

Comment:
when the helper function is added, the typeindex header is in `util/hash.h` and no longer needed here.